docs/whatsnew/4.0.rst

```diff
@@ -31,8 +31,12 @@ Accordingly, the default behavior for these functions has been changed to use Na
 To obtain the prior behavior, which used zero by default for such pixels, the ``missing`` needs to be explicitly set to zero.
 
 The default rotation function used internally has been switched to :func:`scipy.ndimage.affine_transform` from :func:`skimage.transform.warp` due to changes in the latest release of `skimage`, although the differences between the rotation results are small.
+Since :func:`scipy.ndimage.affine_transform` has more consistent results between biquartic interpolation (``order=4``) and bicubic interpolation (``order=3``), we have changed the default interpolation order for :meth:`sunpy.map.GenericMap.rotate` to bicubic interpolation for improved performance under default settings.
```

Comment:
I don't understand this sentence, in particular:
> has more consistent results between biquartic interpolation (``order=4``) and bicubic interpolation

What does having consistent results between two methods mean?

Reply:
This is saying that scipy's bicubic looks more like scipy's biquartic than skimage's bicubic looks like skimage's biquartic.  skimage's bicubic is fast, but can have noticeably poor quality.  That's why, when skimage was the default, we switched to biquartic as the default in #1185.

Reply:
Reworded